frontend/src/pages/create_campaign/Preview/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { SubmitButton, SubmitWrapper } from "./campaignPreview.styled";
+import ApplicationForm from "../../../components/ApplicationForm";
+
+// FIXME: CHAOS-65, add campaign card to preview tab
+const CampaignReview = (props) => {
```

Comment:
deconstruct props (just like above) - this removes the need for the props variable